en/rocky/8/guides/rocky-8-installation.md

```diff
@@ -68,159 +67,190 @@ Insert and boot off the installation medium (optical disk, USB flash drive, and
+- _Software_: (Installation Source and Software Selection)
+- _System_: (Installation Destination and Network & Hostname)
+
+We delve into each of these sections next and make changes where necessary.
 
 ## Localization Section
-This section is used for customizing items related to the locale of the system. This includes – Keyboard, Language Support, Time and Date.
 
+This section is used for customizing items related to the locale of the system. This includes – Keyboard, Language Support, Time and Date.
 
 ### Keyboard
-On our demo system in this guide, we accept the default value (English US) and make no changes.
-However if you need to make any changes here, from the Installation Summary screen, click the Keyboard option to specify the keyboard layout of the system. You can add additional keyboard layouts if you need to in the ensuing screen and specify their order.
-Click Done when you are done.
+
+On our demo system in this guide, we accept the default value (_English US_) and make no changes.
+
+However if you need to make any changes here, from the _Installation Summary_ screen, click the <kbd>Keyboard</kbd> option to specify the keyboard layout of the system. You can add additional keyboard layouts if you need to in the ensuing screen and specify their order.
+
+Click <kbd>Done</kbd> when you are done.
 
 ### Language Support
-The Language Support option on the Installation Summary screen enables you to specify support for additional languages that you may need on the finished system.
-We accept the default value (English – United States) and make no change. Click Done.
 
+The <kbd>Language Support</kbd> option on the _Installation Summary_ screen enables you to specify support for additional languages that you may need on the finished system.
+
+We accept the default value (__English – United States__) and make no change. Click <kbd>Done</kbd>.
 
 ### Time & Date
-Click the Time & Date option on the main Installation Summary screen to bring up another screen that will allow you to select the time zone in which the machine is located. Scroll through the list of regions and cities and select the area closest to you. 
 
-Depending on your installation source, the Network Time option could be set to ON or OFF by default. Accept the default ON setting; this allows the system to automatically set the correct time using the Network Time Protocol (NTP). Click Done after making any changes.
+Click the <kbd>Time & Date</kbd> option on the main _Installation Summary_ screen to bring up another screen that will allow you to select the time zone in which the machine is located. Scroll through the list of regions and cities and select the area closest to you.
+
+Depending on your installation source, the _Network Time_ option could be set to _ON_ or _OFF_ by default. Accept the default _ON_ setting; this allows the system to automatically set the correct time using the Network Time Protocol (NTP). Click <kbd>Done</kbd> after making any changes.
 
```

Comment:
Either of these works fine. The formatting differences that you are proposing should be reviewed by @wsoyinka since he wrote the document.